package.json

```diff
@@ -26,7 +26,7 @@
     "grunt-cli": "1.2.0",
     "grunt-rev": "0.1.0",
     "load-grunt-tasks": "3.5.2",
-    "mozlog": "2.0.6",
+    "mozlog": "2.2.0",
```

Comment:
Do you want this change?

Reply:
Yeah, there is a merge `npm audit` warning otherwise

Reply:
I updated the commit message